build/args/testing.gn

```diff
@@ -6,8 +6,6 @@ is_official_build = false
 dcheck_always_on = true
 symbol_level = 1
 
-strip_absolute_paths_from_debug_symbols = false
```

Comment:
did this option go away? it seems like we should still be setting this.

Reply:
It went away

Reply:
https://chromium-review.googlesource.com/c/chromium/src/+/1898778 , check the chromium thread linked in the commit, adding this as default is a bad idea. There seems to be other ways to do this properly.